README.md

```diff
@@ -40,14 +40,21 @@ The reactive BLE lib supports the following:
 ### Android
 
 You need to add the following permissions to your AndroidManifest.xml file:
+
 ```xml
 <uses-permission android:name="android.permission.BLUETOOTH_SCAN" android:usesPermissionFlags="neverForLocation" />
 <uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
 <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" android:maxSdkVersion="30" />
 <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" android:maxSdkVersion="30" />
 ```
 
-If you use `BLUETOOTH_SCAN` to determine location, remove `android:usesPermissionFlags="neverForLocation"`
+If you use `BLUETOOTH_SCAN` to determine `location` modify your AndroidManfiest.xml file:
```

Comment:
It is already mentioned in Readme, what did you add exactly?

Reply:
`README.md` mentioned remove `android:usesPermissionFlags="neverForLocation"` from user's `AndroidManifest.xml` declaration.  
But  [RxAndroidBLE](https://github.com/dariuszseweryn/RxAndroidBle) library already declared  `android:usesPermissionFlags="neverForLocation"` on there `AndroidManifest.xml`, and it would be [merged]( https://developer.android.com/studio/build/manifest-merge) at last.  
So if you want scan for location, you need to remove it from not only **your** `AndroidManifest.xml` declaration, but **package** `AndroidManifest.xml` declaration.  
And following `xml` is for that

Reply:
I'd suggest to correct this line (in both files; also, why we have the same stuff in two files??) as follows:

```markdown
If you use `BLUETOOTH_SCAN` to determine location, modify your AndroidManfiest.xml file to include the following entry:
```

Could you update the pull request, @keenranger?

Reply:
@werediver, Pull request updated